controllers/home.js

```diff
@@ -2,10 +2,32 @@
  * GET /
  * Home page.
  */
+var Projects = require('../models/Projects')
+
+function getProjects (res) {
```

Comment:
It's repetitive to recreate a getProjects in the home js file. Let me know if there's a more modular way to handle this!